core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala

```diff
@@ -377,6 +384,18 @@ class BaseTiSparkTest extends QueryTest with SharedSQLContext {
       }
     }
 
+    // test tiflash's result
+    if (spark.conf.get(TiConfigConst.ENABLE_TIFLASH_TEST).toBoolean) {
+      // get result from TiFlash
+      try {
+        spark.conf.set(TiConfigConst.USE_TIFLASH, "true")
```

Comment:
why do we need to set this configuration to true?

Reply:
This flag is used to control code behavior. 

The ENABLE_TIFLASH_TEST is to control test behavior.

Reply:
Is there any case that code behavior should be different from test behavior? Could you please describe them in comments for better understanding?